### PR TITLE
Add the doc skeleton for Admin Guide->Basic Concept

### PR DESCRIPTION
**Pull request by daniceexi**
If we all agree with the format, we will add others.

### CODE REVIEW

File: docs/source/guides/admin-guides/basic_concepts/network_planning/network_service/index.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Network Services which are used in xCAT
+==================================
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/basic_concepts/network_planning/network_service/index.rst]: Need to add a few more = , line 2 is too short. Causes a build warning.

File: docs/source/guides/admin-guides/basic_concepts/xcat_object/index.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+xCAT Object
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/basic_concepts/xcat_object/index.rst]: Suggest "xCAT Data Objects"

File: docs/source/guides/admin-guides/basic_concepts/xcat_object/index.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+===========
+
+Node Object
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/basic_concepts/xcat_object/index.rst]: Let's make these all names of the object lower case...  that would make it consistent with the manpage.
node object, osimage object, group object.  I also like putting `` around node to make it formatted text. 

nodeobject

Comment (whowutwut) [inline docs/source/guides/admin-guides/basic_concepts/xcat_object/index.rst]: On 2nd thought, played with the formatting, if we have "xCAT Data Objects", we don't need to use object in each of the objects.  Just list them, node, osimage, group, etc...  

The side index bar is also much cleaner 

Here's my suggestion rendered: 

sidebar

File: docs/source/guides/admin-guides/basic_concepts/xcat_db/index.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
+xCAT Database
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/basic_concepts/xcat_db/index.rst]: I'm going to merge this in "as-is" I have some ideas that I can do to reorg this section but I will create a pull request on top of this

**Comment by whowutwut** [comment]
@daniceexi I merged this in, will continue with some more outline and submit another pull request for you to take over. Maybe not the best way to collaborate, but let's give it a try...

**Comment by daniceexi** [comment]
@whowutwut I made the change base on your commit:d13fd1b

**Pull request merged**